packages/interface/README.md

```diff
@@ -0,0 +1,77 @@
+
+The API exposes a single action `setSingleActiveArea`, and a single selector `getSingleActiveArea`.
+
+The action `setSingleActiveArea` receives a scope and the area that should be active in a given scope or `undefined` if no area should be active (e.g., the area is closed.)
+
+The selector `getSingleActiveArea` receives the scope and returns the area that is active in that scope or `undefined` if no area is active.
+
+Bellow are some examples of the action and selector used together:
+```js
+wp.data.dispatch( 'core/interface' ).setSingleActiveArea( 'edit-site/complementary-area', 'edit-site/block-inspector'  );
+
+wp.data.select( 'core/interface' ).getSingleActiveArea( 'edit-site/complementary-area' ); -> "edit-site/block-inspector"
+
+wp.data.dispatch( 'core/interface' ).setSingleActiveArea( 'edit-site/complementary-area', 'edit-site/global-styles'  );
+
+wp.data.select( 'core/interface' ).getSingleActiveArea( 'edit-site/complementary-area' ); -> "edit-site/global-styles"
+
+wp.data.dispatch( 'core/interface' ).setSingleActiveArea( 'edit-site/complementary-area' );
+
+wp.data.select( 'core/interface' ).getSingleActiveArea( 'edit-site/complementary-area' ); -> undefined
+```
+
+#### Multiple Active Area Functionality
+
+This functionality allows controlling which areas are active in situations where multiple areas can be active at the same time.
+It can be used, for example, to control which panels are open are which items should appear in an area for favorites.
+
+Similar to the single active area the API exposes a single action `setMultipleActiveAreaEnableState`, and a single selector `isMultipleActiveAreaActive`.
+
+The action `setMultipleActiveAreaEnableState` receives a scope, the identifier of the area, and a boolean if true the are should be active if false the area should not be active.
+
+The selector `isMultipleActiveAreaActive` receives the scope, the identifier of the area, and returns true if an area is active and false otherwise.
+Bellow are some examples of the action and selector used together:
+```js
+wp.data.select( 'core/interface' ).isMultipleActiveAreaActive( 'edit-post/complementary-area', 'edit-post/block-patterns-sidebar' ); -> false
+
+wp.data.dispatch( 'core/interface' ).setMultipleActiveAreaEnableState( 'edit-post/complementary-area', 'edit-post/block-patterns-sidebar', true );
+
+wp.data.select( 'core/interface' ).isMultipleActiveAreaActive( 'edit-post/complementary-area', 'edit-post/block-patterns-sidebar' ); -> true
+```
+
+
+### Components
+
+Four components are exposed. The components are based on slot & fill paradigm two components are "fills" and two components are "slots".
+
+`ComplementaryArea` and `ComplementaryArea.Slot` form a slot fill pair to render complementary areas. Multiple `ComplementaryArea` components representing different complementary areas may be rendered at the same time, but only one appears on the slot depending on which complementary area is enabled.
```

Comment:
`ComplementaryArea` feels like a weird name to me. I don't have a better name though. Thoughts @aduth @mtias

Reply:
`complementary` is one of the ARIA regions:
https://www.w3.org/TR/wai-aria-1.1/#region

> Authors SHOULD limit use of the region role to sections containing content with a purpose that is not accurately described by one of the other `landmark` roles, such as `main`, `complementary`, or `navigation`.

More details:

https://www.w3.org/TR/wai-aria-1.1/#complementary

https://www.w3.org/TR/wai-aria-practices/examples/landmarks/complementary.html

Reply:
For the full picture, it's based on previous discussions that using `Sidebar` as the component name is wrong here because it behaves differently on mobile web (modal like) and native mobile (bottom sheet). In addition, we should try to avoid naming high-level components after their placement on the screen but rather phrase them in a way that expresses their intent, some good examples that exist:
- `BlockControls`
- `InsepectorControls`
- `PinnedItems`